custom/bin/ARE/data/webservice/angular/envControl/component/help/faq/html/faq3en.html

```diff
@@ -1,9 +1,11 @@
-<p>EN Es kann nicht verallgemeinert werden, welche zusätzliche Hardware erworben werden muss. Dies hängt davon
-    ab, welche Geräte mit AsTeRICS Ergo gesteuert werden sollen.
-    Um herauszufinden, welche zusätzliche Hardware gekauft werden muss, verwenden Sie den
-    <a href="" ng-click="$ctrl.toState('asterics.envControl.STATE_HELP_DEVICES')">Hardware-Assistenten</a>.
-    Wählen sie alle Geräte, die gesteuert werden sollen, aus. Die
-    Hardware, die Sie zur Steuerung benötigen, wird darunter angezeigt.
-
+<p>The required hardware depends on the devices, which you wish to control with AsTeRICS Ergo.
+    Use the 
+    <a href="" ng-click="$ctrl.toState('asterics.envControl.STATE_HELP_DEVICES')">Order Wizard</a>.
+   to find out which additional hardware you need to acquire.
+1. If you want to control a device, which you normally control using a remote control, you need to acquire an IrTrans. The IrTrans allows to replace remote commands. One IrTrans can control many remote control devices. Therefore you only need to acquire one IrTrans. No further hardware is needed. Please note that you also need to order a USB Type B cable, as this is not included in the order.
```

Comment:
same as before, better use <ol>

Reply:
see https://github.com/asterics/AsTeRICS-Ergo/pull/62/files how to use `<ol>`. it is a "ordered list" and HTML does generate the numbers automatically.